kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java

```diff
@@ -932,7 +940,17 @@ private void initForm() {
         });
         this.security.addMouseOutHandler(event -> resetHelp());
         for (GwtWifiSecurity mode : GwtWifiSecurity.values()) {
+
             this.security.addItem(MessageUtils.get(mode.name()));
+            /**
```

Comment:
This needs to be removed, here for testing purposes

Reply:
commented code is a mechanism to prevent the Enterprise for showing on for net 1 and ap mode